### ISSUE TITLE
Push version 1.2 to Pypi?

### ISSUE THREAD

**Issue by Mikea1985** [fraserw/trippy#39]
After a collaborator tried to install Trippy (from Pypi, not from GitHub) and was having issues, I looked into this. The following reproduces the procedure and error that my collaborator got: 

    conda create -n trippyconda37 python=3.7 numpy stsci matplotlib astropy sextractor emcee numba pytest
    conda activate trippyconda37
    pip install trippy
    python -c "import numpy, scipy, matplotlib, astropy, emcee, numba"
    python -c "import trippy; from trippy import bgFinder, psfStarChooser, scamp, MCMCfit"
    cd
    git clone https://github.com/fraserw/trippy.git
    cd trippy/trippy/tests
    python tests.py

Which results in the following error: 
[fail.txt](https://github.com/fraserw/trippy/files/4712993/fail.txt)

The same happened if I tried with Python 3.6 rather than 3.7. I believe that what the error message means is that a numpy float is used as an integer index, which is of course a no-no now that it's apparently no longer obvious how to turn a float into an integer. When I remove the pip installed trippy and instead soft-link to the version in the git directory, it all works. Turns out pip is still installing version 1.1.1 whereas here on GitHub we are at 1.2. I assume that the problem with 1.1.1 is only relevant for some versions of Numpy or Python, but rather than looking into that, perhaps its easier to just push the version that does work to Pypi?

**Comment by fraserw** [comment]
Thanks for the heads up! I agree uploading a new version to pip is a good idea. Will do so in the next day or so, and let you know when its done.

**Comment by Mikea1985** [comment]
I think I remember there being an outstanding pull request from me. In not
sure whether that should be integrated before pushing to pip.

**Comment by fraserw** [comment]
Got a number? I thought I had sorted them all...

**Comment by Mikea1985** [comment]
Pull request #36 seems to still be open. It exposes the ability to MCMC fit different parameters independently more easily, and explains it in the tutorial.

**Comment by fraserw** [comment]
Yep that’s right, because I haven’t had time to look at TRIPPy for quite a while...
